### DISCUSSION TITLE
What's the difference between "layers" and "groups"?

### DISCUSSION THREAD

**Discussion by pgilfernandez** [friction2d/friction#306] [Q&A] [answered]
Easy question and probably easy answer but I could not get the difference and when is better to use a group and when to promote it to a layer...

Thanks

**Comment by rodlie** [comment, marked as answer]
Groups are just groups. Layers are used for compositing etc, and the end result is rasterized. If you just work in vector and export SVG you can ignore layers, with masking as the only exception.
